content/influxdb/v2.0/upgrade/v1-to-v2/docker.md

```diff
@@ -0,0 +1,245 @@
+For information about exporting and migrating CQs to tasks, see
+[Migrate continuous queries to tasks](/influxdb/v2.0/upgrade/v1-to-v2/migrate-cqs/).
+{{% /note %}}
+
+## Upgrade requirements
+InfluxDB 2.x provides a 1.x-compatible API, but expects a different storage layout on disk.
+To account for theses differences, the InfluxDB Docker image contains extra functionality to migrate
+1.x data and configuration into 2.x layouts automatically before booting the `influxd` server.
+
+- [InfluxDB 2.x initialization credentials](#influxdb-2x-initialization-credentials)
+- [File system mounts](#file-system-mounts)
+- [Upgrade initialization mode](#upgrade-initialization-mode)
+
+{{% note %}}
+The automated upgrade process will not run if an existing boltdb file is found at the configured path.
+This behavior allows for the InfluxDB container to reboot post-upgrade without overwriting migrated data.
+{{% /note %}}
+
+Find more information about the automated InfluxDB upgrade process,
+see [Upgrade from InfluxDB 1.x to 2.0](/influxdb/v2.0/upgrade/v1-to-v2/).
+
+### InfluxDB 2.x initialization credentials
+The automated InfluxDB upgrade process bootstraps an initial admin user,
+[organization](/influxdb/v2.0/reference/glossary/#organization), and
+[bucket](/influxdb/v2.0/reference/glossary/#bucket) required by InfluxDB 2.x.
+Use the following environment variables to provide setup credentials:
+
+-	`DOCKER_INFLUXDB_INIT_USERNAME`: username to set for the admin user ({{< req >}}).
+-	`DOCKER_INFLUXDB_INIT_PASSWORD`: password to set for the admin user ({{< req >}}).
+-	`DOCKER_INFLUXDB_INIT_ORG`: name to set for the initial organization ({{< req >}}).
+-	`DOCKER_INFLUXDB_INIT_BUCKET`: name to set for the initial bucket ({{< req >}}).
+-	`DOCKER_INFLUXDB_INIT_RETENTION`: duration for the initial bucket's retention period
+  If not set, the initial bucket will retain data forever.
+-	`DOCKER_INFLUXDB_INIT_ADMIN_TOKEN`: The authentication token to associate with the admin user.
+  If not set, InfluxDB auto-generates a token.
+
+### File system mounts
+The InfluxDB upgrade process requires extra volumes to be mounted into the 2.x container.
+Use **environment variables** and **Docker mounts** to specify and configure the
+appropriate mount paths for the following:
+
+-	1.x data on disk
+-	Custom 1.x configuration file (if any)
+
+The InfluxDB upgrade process searches for mounted 1.x data and configuration files
+in the following priority order:
+
+1.	1.x configuration file specified by the `DOCKER_INFLUXDB_INIT_UPGRADE_V1_CONFIG` environment variable
+2.	1.x data directory specified by the `DOCKER_INFLUXDB_INIT_UPGRADE_V1_DIR` environment variable
+3.	1.x configuration file mounted at `/etc/influxdb/influxdb.conf`
+4.	1.x data directory mounted at `/var/lib/influxdb`
+
+{{% note %}}
+#### Avoid data loss
+The automated upgrade process generates both data and configuration files, by default,
+under `/var/lib/influxdb2` and `/etc/influxdb2`.
```

Comment:
```suggestion
By default, the automated upgrade process generates both data and configuration files
under `/var/lib/influxdb2` and `/etc/influxdb2`.
```

Reply:
IMO "by default" implies that it can be disabled, but it can't.